examples/delegator/lib.rs

```diff
@@ -18,21 +14,8 @@ mod delegator {
     /// and in `Subber` state will delegate to the `Subber` contract.
     ///
     /// The initial state is `Adder`.
-    #[derive(
-        Debug,
-        Copy,
-        Clone,
-        PartialEq,
-        Eq,
-        scale::Encode,
-        scale::Decode,
-        SpreadLayout,
-        PackedLayout,
-    )]
-    #[cfg_attr(
-        feature = "std",
-        derive(::scale_info::TypeInfo, ::ink_storage::traits::StorageLayout)
-    )]
+    #[ink_lang::storage_item]
+    #[derive(Debug, Copy, Clone, PartialEq, Eq)]
```

Comment:
Why did you add the `Debug` derives?

Reply:
It was before=) 
<img width="1138" alt="image" src="https://user-images.githubusercontent.com/18346821/169816636-906b61fa-9440-48fd-aa35-4acdc8c1f404.png">